core/src/main/java/com/flipkart/gjex/core/filter/grpc/GrpcAccessLogGjexGrpcFilter.java

```diff
@@ -0,0 +1,45 @@
+package com.flipkart.gjex.core.filter.grpc;
+
+import com.flipkart.gjex.core.filter.RequestParams;
+import com.flipkart.gjex.core.filter.ResponseParams;
+import com.flipkart.gjex.core.logging.Logging;
+import com.google.protobuf.GeneratedMessageV3;
+import io.grpc.Metadata;
+
+/**
+ * Filter for logging grpc access log requests
+ * @author ajay.jalgaonkar
+ *
+ */
+public class GrpcAccessLogGjexGrpcFilter<R extends GeneratedMessageV3,
```

Comment:
Too much-repeated names?`GrpcAccessLogFilter` should have been enough.

Reply:
done